ide-support/revsblibrary.livecodescript

```diff
@@ -2217,6 +2217,30 @@ command revSBAddLibraryMapping pPlatform, pName, pLocation, @xSettingsA
    appendToStringList tMapping, xSettingsA[pPlatform & ",library"]
 end revSBAddLibraryMapping
 
+private function __isEngineModule pID
```

Comment:
@livecodepanos Rather than duplicating the code here, I recommend that we rename the __extensionIsBuiltin private function in extension utils to: `extensionIsBuiltin`, rename the uses of it in the extension-utils script library and use that in the sb library. This means we only have one place to update should we increase the builtin modules.